h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -376,9 +368,23 @@ public AbstractPrediction predict(RowData data, ModelCategory mc) throws Predict
   public AbstractPrediction predict(RowData data) throws PredictException {
     return predict(data, m.getModelCategory());
   }
+  
+  RowToRawDataConverter makeRowDataConverter(Config config, ErrorConsumer errorConsumer) {
+    CategoricalEncoding categoricalEncoding = config.getUseExternalEncoding() ? CategoricalEncodings.AUTO : m.getCategoricalEncoding();
+
+    RowToRawDataConverter rowDataConverter;
+    if (m instanceof RowConverterFactory) {
```

Comment:
I think we can get rid of RowConverterFactory as it adds no value, you can do just `instanceof MojoModel` here and achieve the same result, also you can remove the underscore from the method name, the original method was public as well and did not need underscore

Reply:
ok, I'll do that. I thought people would have more concerns about an explicit casting than about a underscore...

Reply:
well, now realizing that `MojoModel` is abstract, so I agree that interface can be removed